specification/cognitiveservices/data-plane/CustomSearch/stable/v1.0/CustomSearch.json

```diff
@@ -103,7 +103,7 @@
 						"description": "The identifier for the custom search configuration",
 						"required": true,
 						"type": "integer",
-						"format": "int32"
+						"format": "int64"
```

Comment:
@q3blend your intention is to mimic uint here, so it will be better if you also take advantage of minimum/maximum attributes.